osgar/tools/calibfish.py

```diff
@@ -0,0 +1,84 @@
+import cv2
+import glob
+from pprint import pprint
+import numpy as np
+
+PATTERN = (8,5)
+
+def capture(camera=0):
+    cap = cv2.VideoCapture(camera)
+    if not cap.isOpened():
+        return
+
+    print("backend:", cap.getBackendName())
+    cap.set(cv2.CAP_PROP_FRAME_WIDTH, 1280)
```

Comment:
Is this for Kayeton cameras? (i.e. can this code be used for Eduro and Kloubak with Arecont IP cameras too?)

Reply:
Well it could. But someone would have to test it and fix what does not work with them. Currently it is tested with two cameras - Kayeton and my notebook camera (and its default resolution is something really small).